include/cassandra.h

```diff
@@ -9477,6 +9477,12 @@ cass_timestamp_gen_server_side_new();
  * exceeded then a warning is logged and timestamps stop incrementing until the next
```

Comment:
The above comment-block probably needs to be updated -- the new code has proper microsecond precision and bumps the timestamp by 1 when there is clock skew, rather than using the lowest 3 digits as a counter?